docs/kareem/DEDM-HR/usecase.md

```diff
@@ -0,0 +1,157 @@
+# Jupyter Notebook for DEDM-HR
+
+## Jupyter Notebook for Database-Enabled Design Module for High-Rise Buildings
+
+Dae Kun Kwon - [NatHaz Modeling Laboratory](https://nathaz.nd.edu/) & [Center for Research Computing (CRC)](https://crc.nd.edu/), University of Notre Dame<br/>
+Ahsan Kareem - [NatHaz Modeling Laboratory](https://nathaz.nd.edu/), University of Notre Dame<br/>
+
+Key Words: Database-enabled design, High-rise buildings, Wind loads, Wind response
+
+## Resources
+The example makes use of the following DesignSafe resources:<br/>
+• [DesignSafe Data Depot: Jupyter Notebook for DEDM-HR](https://www.designsafe-ci.org/data/browser/public/designsafe.storage.community/Use%20Case%20Products/DEDM-HR/)<br/>
```

Comment:
Please point to specific notebooks and use this type of a link so it will open directly in Jupyter:
`https://jupyter.designsafe-ci.org/hub/user-redirect/notebooks/CommunityData/Use%20Case%20Products/MPM/LearnMPM/point-axial-vibration1d.ipynb`

Reply:
Thank you for your comment @kks32 
Please note that it was intentional to prevent from executing our Notebooks directly via Jupyter hub address. 
We think that the use of direct link via Jupyter hub is for user to run the Notebook easily. However, if we use such a hub link, our Notebooks will throw error when being executed in there. It is because the Notebooks involve data storing from external server in the process but the directory they are currently located is under Community Data, which has a strict restriction, allowing authorized users only. For this reason, we used directory link so that a user first copies those Notebooks to their own Data Depot, then execute it in there, which would not lead to an error: we described this in the usecase content.

Reply:
This is because you are writing output to community data. Instead, you could make the notebook write to `MyData`.

You can handle the location of `MyData` doing something along these lines:

```python
cur_dir = os.getcwd()
if ('jupyter/MyData' in cur_dir ):
    cur_dir = cur_dir.split('MyData').pop() 
    storage_id = 'designsafe.storage.default'
    input_dir = ag.profiles.get()['username']+ cur_dir
    input_uri = 'agave://{}/{}'.format(storage_id,input_dir)
    input_uri = input_uri.replace(" ","%20")
elif('jupyter/mydata' in cur_dir ):
    cur_dir = cur_dir.split('mydata').pop()
    storage_id = 'designsafe.storage.default'
    input_dir = ag.profiles.get()['username']+ cur_dir
    input_uri = 'agave://{}/{}'.format(storage_id,input_dir)
    input_uri = input_uri.replace(" ","%20")    
elif('jupyter/MyProjects' in cur_dir):
    cur_dir = cur_dir.split('MyProjects/').pop()
    PRJ = cur_dir.split('/')[0]
    qq = {"value.projectId": str(PRJ)}
    cur_dir = cur_dir.split(PRJ).pop()
    project_uuid = ag.meta.listMetadata(q=str(qq))[0]["uuid"]
    input_dir = cur_dir
    input_uri = 'agave://project-{}{}'.format(project_uuid,cur_dir)
    input_uri = input_uri.replace(" ","%20")
elif('jupyter/projects' in cur_dir):
    cur_dir = cur_dir.split('projects/').pop()
    PRJ = cur_dir.split('/')[0]
    qq = {"value.projectId": str(PRJ)}
    cur_dir = cur_dir.split(PRJ).pop()
    project_uuid = ag.meta.listMetadata(q=str(qq))[0]["uuid"]
    input_dir = cur_dir
    input_uri = 'agave://project-{}{}'.format(project_uuid,cur_dir)
    input_uri = input_uri.replace(" ","%20")
elif('jupyter/CommunityData' in cur_dir):
    cur_dir = cur_dir.split('jupyter/CommunityData').pop() 
    input_dir = cur_dir
    input_uri = 'agave://designsafe.storage.community/{}'.format(input_dir)
    input_uri = input_uri.replace(" ","%20")    

inputs = {"inputDirectory": [ input_uri ]}
```

Reply:
Thank you for your information @kks32. We definitely didn't know the path (directory structure) of DesignSafe's Data Depot, so your info would be useful later. But we think that it is better to keep the current Notebook due to two reasons: first, it is more intuitive and convenient for user to save the data to the directory where the Jupyter Notebook is. Second,  our Notebook offers a flexibility to change various inputs by user, thus the Notebook itself needs be located in user's MyData so that it can be easily editable, saved, and even copied to other (file)name for further usage.

Anyway, thank you for your kind attention on this and sharing valuable information 👍

Reply:
I am wondering if we can find a middle ground here.  I want to keep the bar very low for folks to find and see the use case notebooks, that is why we want a link in the Documentation that takes them right to Jupyter to see the notebook.  Then they can scroll through, etc.  But then you can make it VERY clear that to use the notebook they need to use the Save As function to save it to MyData. I was able to do that (although the message window when you click Save As is VERY unclear at the moment).  What do you think??

Reply:
Hi @erathje71 Thank you for your suggestion. If we understand correctly, you want us to add a note around the top of the Jupyter Notebook like Save As you mentioned. Just one issue we have is that the Community Data/Use Case Products directory where our DEDM-HR notebooks are somehow does not allow us deleting or editing files, thus we cannot proceed: we did not try to overwrite file, though. Is there any way to remove files in the directory? Then, we can copy new revised files (notebooks) in there.

Reply:
Hi @kks32 (cc: @erathje71), 

Following @erathje71's suggestion, we have added a note in our Notebook files (please see the screenshot below), and changed our usecase.md to directly link to each Notebook, which can be seen in the following link: https://deploy-preview-46--ds-use-case-preview.netlify.app/kareem/dedm-hr/usecase/

Thank you for your helps!

![DEDM-HR note01](https://github.com/DesignSafe-CI/DS_Use_Case_template/assets/12802305/294e4008-17ea-477e-8fb7-a6381017e1d7)